pkg/artifacts/paths.go

```diff
@@ -41,6 +41,20 @@ func BaseDir() string {
 	return d
 }
 
+// RunDir return the path to the directory used for storing all files in general
+// specific to a single run of kubetest2
+func RunDir() string {
+	d := runDir
+	if d == "" {
```

Comment:
I see this was copied from above, but since `d` is being defaulted to s string type, is it not always ""?

Reply:
d fetches its value from `runDir` variable, which indeed is assigning the value passed to flag `--rundir`

Ref: https://github.com/kubernetes-sigs/kubetest2/pull/183/files#diff-76b7e3fd92da504af77a74d8e8e2f888630ee1005503eb1feb04eb30e9b5c172R107

Thus the default value of `d` is not always "". It is "" only when no value is passed to it while running `kubetest`.